src/main/java/org/hyperskill/musicadvisor/Menu.java

```diff
@@ -0,0 +1,64 @@
+package org.hyperskill.musicadvisor;
+
+import org.hyperskill.musicadvisor.commands.*;
+
+import java.util.Scanner;
+
+public class Menu {
+    private boolean workInProcess;
+    private boolean isAuthorized;
+
+    public Menu() {
+        workInProcess = true;
+        isAuthorized = false;
+        work();
+    }
+
+    private void work() {
+        System.out.println("Welcome to the App!");
+        listCommands();
+        while(workInProcess) {
+            String command = getCommand();
```

Comment:
Method getCommand() will create a new Scanner for each iteration of the loop. This is not good